src/rez/plugin_managers.py

```diff
@@ -201,9 +176,55 @@ def load_plugins(self):
                         traceback.print_exc(file=out)
                         print_debug(out.getvalue())
 
-            # load config
-            data, _ = _load_config_from_filepaths([os.path.join(path, "rezconfig")])
-            deep_update(self.config_data, data)
+    def load_plugins_from_entry_points(self):
+        if sys.version_info.minor >= 8:
+            from importlib.metadata import entry_points
+        else:
+            from importlib_metadata import entry_points
+
+        discovered_plugins = entry_points(group='rez.plugins')
+        for plugin in discovered_plugins:
+            plugin = plugin.load()
+            plugin_name = plugin.__name__.split('.')[-1]
```

Comment:
I'm not sure to understand why we need to do this. Can't we just use `plugin.name`? For the old plugin style, we needed to do `split('.')[-1]` because we were getting the name from `rezplugins.<type>.<name>`. But with entrypoints, we don't need to split.